PaySlip_DelRosario.java

```diff
@@ -54,21 +54,43 @@ public double getGrossIncome() {
     return grossIncome;
   }
 
-  public double getWithholdingTax() {
+  // public double getWithholdingTax() {
```

Comment:
Since we have source control, you can just delete this and bring it back if you need to, but try not to leave commented out code.